source/extensions/filters/common/lua/lua.h

```diff
@@ -413,8 +428,8 @@ class ThreadLocalState : Logger::Loggable<Logger::Id::lua> {
     std::vector<int> global_slots_;
   };
 
-  ThreadLocal::SlotPtr tls_slot_;
-  uint64_t current_global_slot_{};
+  absl::flat_hash_map<std::string, ThreadLocal::SlotPtr> tls_slot_map_;
+  absl::flat_hash_map<std::string, uint64_t> current_global_slot_map_;
```

Comment:
Why not keep ThreadLocalState unchanged, but maintain a map in FilterConfig to manage different Lua scripts?